cmake/developer_package/frontends/frontends.cmake

```diff
@@ -141,8 +141,13 @@ macro(ov_add_frontend)
     # Disable all warnings for generated code
     set_source_files_properties(${PROTO_SRCS} ${PROTO_HDRS} PROPERTIES COMPILE_OPTIONS -w GENERATED TRUE)
 
+    # Shutdown protobuf when unloading the front dynamic library
+    if(OV_FRONTEND_SHUTDOWN_PROTOBUF AND BUILD_SHARED_LIBS)
```

Comment:
I don't see the definition of `OV_FRONTEND_SHUTDOWN_PROTOBUF` in this PR

Reply:
It's the option defined in #Line 92